### PR TITLE
Fixed Initialization Race Condition in DisposeHelperTest

### PR DESCRIPTION
**Pull request by nixon124**
Fixed an initialization race condition that occurred in DisposeHelperTest class.

### CODE REVIEW

File: src/test/java/emissary/util/DisposeHelperTest.java [hunk 1]
```diff
@@ -22,6 +23,9 @@
 
 class DisposeHelperTest {
 
+    static {
```
Comment (drivenflywheel) [inline src/test/java/emissary/util/DisposeHelperTest.java:26]: This solution works, but we should avoid using a static class initialization block and instead use the common JUnit pattern of performing "before all tests" setup in a static method that's annotated with the `org.junit.jupiter.api.BeforeAll` annotation.  _(We should also move this new method to be just above the existing `setup()` method.)_

```suggestion
    @BeforeAll
    static void setupClass() {
```

**Review by drivenflywheel — CHANGES_REQUESTED** [review]
Minor change requested to follow common JUnit practices.

File: src/test/java/emissary/util/DisposeHelperTest.java (hunk 1 above)
Comment (nixon124) [reply src/test/java/emissary/util/DisposeHelperTest.java]: Got it.

**Review by drivenflywheel — DISMISSED** [review]
🚀

**Review by fbruton — CHANGES_REQUESTED** [review]
Suggest extending `UnitTest`.

**Review by fbruton — APPROVED** [review]

**Review by drivenflywheel — APPROVED** [review]
🚀

**Pull request merged**